src/scss/components/_listings.scss

```diff
@@ -6,20 +6,21 @@
  *
  */
 
-$listing-background         : $white;
-$listing-promoBackground    : $orange;
-$listing-borderColor        : $color-border;
-$listing-iconFill           : $grey--mid;
-$listing-labelBackground    : $grey--offWhite;
-$listing-offlineBorderColor : $grey--lighter;
-$listing-offlineBackground  : $grey--lightest;
-$listing-borderRadius       : 4px;
+$listing-background             : $white;
+$listing-promoLabelTextColor    : $purple;
+$listing-promoLabelBackground   : $purple--light;
+$listing-borderColor            : $color-border;
+$listing-iconFill               : $grey--mid;
+$listing-labelBackground        : $grey--offWhite;
+$listing-labelBorder            : 2px;
+$listing-offlineBackground      : $grey--lightest;
+$listing-borderRadius           : 4px;
+$listing-img-borderRadius       : 2px;
+$listing-textDividerBackground  : $grey--light;
+$listing-promotionColor         : $orange;
```

Comment:
Should probably be called `listing-promoIconColor` as otherwise this variable is a little vague

Reply:
fixed 👍